example/tensorflow-notebook.yaml

```diff
@@ -44,12 +44,11 @@ spec:
       - key: nvidia.com/gpu
         effect: NoSchedule
       securityContext:
-        supplementalGroups:
-        - 0
+        runAsUser: 0
```

Comment:
It needs to run as privileged?

Reply:
Good catch, I did not mean to check that in.

Reply:
Fixed PTAL